ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/PeerReputationTest.java

```diff
@@ -24,8 +24,14 @@
 
 public class PeerReputationTest {
 
+  private static final int INITIAL_SCORE = 25;
   private static final int MAX_SCORE = 50;
-  private final PeerReputation reputation = new PeerReputation(MAX_SCORE);
+  private final PeerReputation reputation = new PeerReputation(INITIAL_SCORE, MAX_SCORE);
+
+  @Test(expected = java.lang.IllegalArgumentException.class)
+  public void shouldThrowOnInvalidInitialScore() {
+    new PeerReputation(2, 1);
```

Comment:
this should be 1, 2

Reply:
the wrong order is intented to test that initial must not be greather than max